docs/conf.py

```diff
@@ -65,6 +66,19 @@
 link_str = f" https://github.com/Qiskit/qiskit-ibm-runtime/blob/stable/{vers[0]}.{vers[1]}/docs/"
 nbsphinx_prolog += link_str + "{{ docname }}"
 
+# ----------------------------------------------------------------------------------
+# Intersphinx
+# ----------------------------------------------------------------------------------
+
+intersphinx_mapping = {
+    "rustworkx": ("https://www.rustworkx.org/", None),
+    "qiskit": ("https://docs.quantum.ibm.com/api/qiskit/", None),
+    "qiskit-aer": ("https://qiskit.github.io/qiskit-aer/", None),
+    "numpy": ("https://numpy.org/doc/stable/", None),
+    "matplotlib": ("https://matplotlib.org/stable/", None),
+    "python": ("https://docs.python.org/3/", None),
```

Comment:
Imo we can get rid of this one. It results in the docs being a bit noisy linking to primitive types like `list`

```suggestion
```